.github/workflows/build.yml

```
@@ -26,9 +26,15 @@ jobs:
       run: dotnet restore
     - name: Build
       run: dotnet build --configuration Release --no-restore
+    - name: Version
+      run: |
+        MOD_VERSION=$(grep -oP '<Version>\K[^<]+' BlasII.CheatConsole/BlasII.CheatConsole.csproj)
+        echo "MOD_VERSION=$MOD_VERSION" >> $GITHUB_ENV
+        BUILD_DATE=$(date +'%Y%m%d%H%M%S')
```

```suggestion
        BUILD_DATE=$(date +'%Y-%m-%d')
```
Use dashes to separate date (Not actually sure if this works)